README

```diff
@@ -165,38 +162,27 @@ will _not_ need to be re-sealed.
 
 a) SINIT module
 
-You should download the SINIT module required for your system.
-
-Intel documented the required SINIT module depending on your CPU platform in:
-http://software.intel.com/en-us/articles/intel-trusted-execution-technology
-But DO NOT download the modules besides the SINIT/RACM from here. The download links provide old versions and broken binaries.
-
-You can then download the module and unzip it. All the modules could be
-downloaded from:
-
-https://cdrdv2.intel.com/v1/dl/getContent/630744?wapkw=intel%20txt%20sinit%20acm%20revocation%20
-
-Find the module fitting to your platform and rename it to the names mentioned on the intel website link.
+You should download the SINIT module required for your system. All the modules
+could be downloaded from:
 
-Also, make sure you have the latest RACM update, if available (2nd & 3rd gen):
-https://software.intel.com/system/files/article/183305/intel-txt-sinit-acm-revocation-tools-guide-rev1-0_2.pdf
+https://cdrdv2.intel.com/v1/dl/getContent/630744?explicitVersion=true
 
-It's possible to use 3rd gen SINIT/RACM on 2nd gen platforms. In fact, the
-only RACM available at the time of writing is for the 3rd gen, while the 2nd
-gen platforms were also affected by the buffer overflow bug in old SINIT
-version.
-
-Finally, you should retrieve the BIN file inside /boot in dom0. E.g., run from
-dom0:
+Find the module fitting to your platform using the list included in downloaded
+archive. Finally, you should retrieve the BIN file inside /boot in dom0. E.g.,
+run from dom0:
 
 $ sudo -s
 # qvm-run --pass-io vm_name_containing_bin_file 'cat /home/user/path_to_sinit/name_of_sinit_file.BIN' > /boot/name_of_sinit_file.BIN
 
+It is best to keep original file name, both because GRUB configuration script
+searches for specific pattern and because version number is included in the
```

Comment:
```suggestion
searches for specific pattern (`*_sinit_*.bin`, case ignored) and because version number is included in the
```

Reply:
Changed in https://github.com/TrenchBoot/qubes-antievilmaid/compare/74a1e4d3162389dcabe65f3e6781e56238ba327f..c2b7a55520af9571a84c8ebbfe6565e94ea230b5, dropped underscores as they are not part of the pattern.